commit 7af0f69e0ce0e589c8acd33b642861f54785b746 (refs/changes/23/3723/2)
Author: Josh Wilsdon <josh@wilsdon.ca>
Date:   2018-03-28T16:27:07-07:00 (1 year, 6 months ago)
    
    TRITON-274 joyent-schemas should support bhyve sysinfo keys
    Reviewed by: Joshua M. Clulow <jmc@joyent.com>
    Reviewed by: Orlando Vazquez <orlando@joyent.com>
    Approved by: Orlando Vazquez <orlando@joyent.com>

diff --git a/lib/cnapi/server.js b/lib/cnapi/server.js
index 40e78c6..14addaa 100644
--- a/lib/cnapi/server.js
+++ b/lib/cnapi/server.js
@@ -1,5 +1,5 @@
 /*
- * Copyright 2014 Joyent, Inc.  All rights reserved.
+ * Copyright 2018 Joyent, Inc.  All rights reserved.
  */
 
 var schema_util = require('../util.js');
@@ -199,6 +199,8 @@ var s_sysinfo = {
 		'HW Family': base.str.req,
 		'Setup': base.bool.req,
 		'VM Capable': base.bool.req,
+		'Bhyve Capable': base.bool.opt,
+		'Bhyve Max Vcpus': base.intg.pos.opt,
 		'CPU Type': base.str.nonempty.req,
 		'CPU Virtualization': base.str.req,
 		'CPU Physical Cores': base.intg.pos.req,
diff --git a/lib/dapi/server.js b/lib/dapi/server.js
index 6e95b81..9d376fb 100644
--- a/lib/dapi/server.js
+++ b/lib/dapi/server.js
@@ -1,5 +1,5 @@
 /*
- * Copyright 2014 Joyent, Inc.  All rights reserved.
+ * Copyright 2018 Joyent, Inc.  All rights reserved.
  */
 
 var schema_util = require('../util.js');
@@ -47,6 +47,8 @@ var s_si_nics_r = schema_util.qualify(s_si_nics, qual.U.req);
 var s_sysinfo = {
 	'type': 'object',
 	'properties': {
+		'Bhyve Capable': base.bool.opt,
+		'Bhyve Max Vcpus': base.intg.pos.opt,
 		'CPU Total Cores': base.intg.pos.req,
 		'Network Interfaces': s_si_nics_r,
 		'Live Image': base.str.nonempty.req,
diff --git a/package.json b/package.json
index abb191c..51bfe6f 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
 	"name": "joyent-schemas",
 	"description": "Joyent JSON Schemas",
-	"version": "0.2.0",
+	"version": "0.3.0",
 	"author": "Joyent (joyent.com)",
 	"main": "lib/index.js",
 	"repository": {
